client/src/app.jsx

```diff
@@ -82,7 +83,7 @@ class App extends React.Component {
   }
 
   componentDidMount() {
-    this.getAllProductInfo('28212')
+    this.getAllProductInfo('29004')
```

Comment:
Rather than hard code this here, we can set the default state in the constructor and use this.state.mainProductId here.

Reply:
oh yeah, definitely! I will change it, but I didn't write this particular code, just changed the number to test some stuff. Will change now!

Reply:
Oh I see you already changed it! i accepted the incoming change and pushed it up!